src/borg/archiver.py

```diff
@@ -1070,6 +1073,11 @@ def _delete_archives(self, args, repository):
                     deleted = True
                     msg = 'Would delete: {} ({}/{})' if dry_run else 'Deleted archive: {} ({}/{})'
                     logger.info(msg.format(archive_name, i, len(archive_names)))
+                    list_msg = 'Would delete archive:' if dry_run else 'Deleted archive:'
+                    if args.output_list:
+                        list_logger.info("{message:<40} {archive}".format(
+                            message=list_msg, archive=format_archive(current_archive)
+                        ))
```

Comment:
hmm, are we duplicating functionality we already have?

it looks like giving `--info` (aka `-v`), see code in 1074/1075, does already what you added for `--list`.

(Sorry, was not aware of that when commenting on the ticket.)

Reply:
--info shows only the name of the archive. Since the timestamp of the archives is also printed when deleting the entire repository, I think it would be more consistent to do this for parts of the repository as well.

Reply:
ok, the timestamp might be useful indeed (at least for people who do not include it into the archive name anyway).

but still, the functionality is now almost duplicate and i think one way should be enough.
if that way shall include the timestamp, then just add that.

Reply:
okay, i'll do that and then make a new PR. :)